From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Thu, 11 Aug 2011 16:43:49 +0200
Subject: man page tweaks

typos in man page nroff formatting

man dash hyphen slash
---
 man/etter.conf.5.in       |   15 +++---
 man/ettercap.8.in         |  129 ++++++++++++++++++++++-----------------------
 man/ettercap_curses.8.in  |   12 ++---
 man/ettercap_plugins.8.in |   50 +++++++++---------
 man/etterfilter.8.in      |   10 ++--
 man/etterlog.8.in         |   66 +++++++++++------------
 6 files changed, 139 insertions(+), 143 deletions(-)

diff --git a/man/etter.conf.5.in b/man/etter.conf.5.in
index da76540..5907f44 100644
--- a/man/etter.conf.5.in
+++ b/man/etter.conf.5.in
@@ -22,8 +22,7 @@
 ..
 .TH ETTER.CONF 5 "" "ettercap @VERSION@"
 .SH NAME
-.B etter.conf 
-\- Ettercap configuration file
+etter.conf \- Ettercap configuration file
 
 .SH DESCRIPTION
 .B "etter.conf"
@@ -96,7 +95,7 @@ IDS) up to the timeout of the ARP cache (which depends on the poisoned operating
 Enable the sending of a spoofed ICMP message to force the targets to make an
 arp request. This will create an arp entry in the host cache, so ettercap
 will be able to win the race condition and poison the target. Useful against
-targets that don't accept gratuitous arp if the entry is not in the cache.
+targets that do not accept gratuitous arp if the entry is not in the cache.
 
 .TP
 .B arp_poison_reply
@@ -223,9 +222,9 @@ in unoffensive mode on a host with more than one network interface
 .TP
 .B checksum_warning
 If you set the value to 0 the messages about incorrect checksums will not be
-displayed in the user messages windows (nor logged to a file with -m). 
+displayed in the user messages windows (nor logged to a file with \-m). 
 .br
-Note that this option won't disable the check on the packets, but only prevent
+Note that this option will not disable the check on the packets, but only prevent
 the message to be displayed (see below). 
 
 .TP
@@ -271,7 +270,7 @@ Here is a list of values: 0 Black, 1 Red, 2 Green, 3 Yellow, 4 Blue, 5 Magenta,
 .TP
 .B utf8_encoding
 specifies the encoding to be used while displaying the packets in UTF-8 format.
-Use the `iconv --list` command for a list of supported encodings.
+Use the `iconv \-\-list` command for a list of supported encodings.
 
 .TP
 .B remote_broswer
@@ -290,7 +289,7 @@ The page requested inside the GET request.
 
 .TP
 .B redir_command_on
-You have to provide a valid command (or script) to enable tcp redirection at
+You must provide a valid command (or script) to enable tcp redirection at the
 kernel level in order to be able to use SSL dissection. Your script should be
 able to get 3 parameters:
 .RS
@@ -305,7 +304,7 @@ etc).
 .B %rport
 The internally bound port to which ettercap listens for connections.
 .RE
-NOTE: this script is executed with an execve(), so you can't use pipes or
+NOTE: this script is executed with an execve(), so you cannot use pipes or
 output redirection as if you were in a shell. We suggest you to make a script if
 you need those commands.
 
diff --git a/man/ettercap.8.in b/man/ettercap.8.in
index 482da32..fb36222 100644
--- a/man/ettercap.8.in
+++ b/man/ettercap.8.in
@@ -22,8 +22,7 @@
 ..
 .TH ETTERCAP "8" "" "ettercap @VERSION@"
 .SH NAME
-.B ettercap @VERSION@ 
-\- A multipurpose sniffer/content filter for man in the middle attacks
+ettercap \- multipurpose sniffer/content filter for man in the middle attacks
 
 .SH ***** IMPORTANT NOTE ******
 Since ettercap NG (formerly 0.7.0), all the options have been changed. Even the target
@@ -35,7 +34,7 @@ specification has been changed. \fIPlease read carefully this man page.\fR
 .PP
 \fITARGET\fR is in the form \fIMAC/IPs/IPv6/PORTs\fR
 .br
-where IPs and PORTs can be ranges (e.g. /192.168.0.1-30,40,50/20,22,25)
+where IPs and PORTs can be ranges (e.g. /192.168.0.1\-30,40,50/20,22,25)
 .SH DESCRIPTION
 Ettercap was born as a sniffer for switched LAN (and obviously even "hubbed" ones),
 but during the development process it has gained more and more features that have
@@ -46,7 +45,7 @@ and includes many features for network and host analysis (such as OS fingerprint
 It has two main sniffing options:
 .PP
 \fIUNIFIED\fR, this method sniffs all the packets that pass on the cable. You can choose to put or not the
-interface in promisc mode (-p option). The packet not directed to the host
+interface in promisc mode (\-p option). The packet not directed to the host
 running ettercap will be forwarded automatically using layer 3 routing. So you
 can use a mitm attack launched from a different tool and let ettercap modify the
 packets and forward them for you. 
@@ -130,26 +129,26 @@ e.g.
 .PP
 \fIMAC\fR must be unique and in the form 00:11:22:33:44:55
 .PP
-\fIIPs\fR is a range of IP in dotted notation. You can specify range with the - (hyphen)
+\fIIPs\fR is a range of IP in dotted notation. You can specify range with the \- (hyphen)
 and single ip with , (comma). You can also use ; (semicolon) to indicate
 different ip addresses.
 .br
 e.g.
 .br
-"10.0.0.1-5;10.0.1.33" expands into ip 10.0.0.1, 2, 3, 4, 5 and 10.0.1.33
+"10.0.0.1\-5;10.0.1.33" expands into ip 10.0.0.1, 2, 3, 4, 5 and 10.0.1.33
 .PP
-\fIPORTs\fR is a range of PORTS. You can specify range with the - (hyphen) and single port 
+\fIPORTs\fR is a range of PORTS. You can specify range with the \- (hyphen) and single port 
 with , (comma).
 .br
 e.g.
 .br
-"20-25,80,110" expands into ports 20, 21, 22, 23, 24, 25, 80 and 110
+"20\-25,80,110" expands into ports 20, 21, 22, 23, 24, 25, 80 and 110
 .PP
 .B NOTE:
 .br
-you can reverse the matching of the TARGET by adding the -R option to the
+you can reverse the matching of the TARGET by adding the \-R option to the
 command line. So if you want to sniff ALL the traffic BUT the one coming or
-going to 10.0.0.1 you can specify "./ettercap -R /10.0.0.1/"
+going to 10.0.0.1 you can specify "./ettercap \-R /10.0.0.1/"
 .PP
 .B NOTE:
 .br
@@ -157,7 +156,7 @@ TARGETs are also responsible of the initial scan of the lan. You can use them
 to restrict the scan to only a subset of the hosts in the netmask. The result
 of the merging between the two targets will be scanned. remember that not
 specifying a target means "no target", but specifying "//" means "all the hosts in
-the subnet.
+the subnet".
 .PP
 
 
@@ -179,16 +178,16 @@ the issuer is modified and signed with the private key contained in the 'etter.s
 file. If you want to use a different private key you have to regenerate this
 file. To regenerate the cert file use the following commands:
 .Sp
-openssl genrsa -out etter.ssl.crt 1024
+openssl genrsa \-out etter.ssl.crt 1024
 .br
-openssl req -new -key etter.ssl.crt -out tmp.csr
+openssl req \-new \-key etter.ssl.crt \-out tmp.csr
 .br
-openssl x509 -req -days 1825 -in tmp.csr -signkey etter.ssl.crt -out tmp.new
+openssl x509 \-req \-days 1825 \-in tmp.csr \-signkey etter.ssl.crt \-out tmp.new
 .br
 cat tmp.new >> etter.ssl.crt
 .br
-rm -f tmp.new tmp.csr
-.SP
+rm \-f tmp.new tmp.csr
+.PP
 NOTE: SSL mitm is not available (for now) in bridged mode.
 
 
@@ -227,7 +226,7 @@ You can choose the mitm attack that you prefer and also combine some of them to
 perform different attacks at the same time.
 .br
 If a mitm method requires some parameters you can specify them after the colon.
-(e.g.  -M dhcp:ip_pool,netmask,etc )
+(e.g.  \-M dhcp:ip_pool,netmask,etc )
 .Sp
 The following mitm attacks are available:
 .RS
@@ -238,8 +237,8 @@ requests/replies are sent to the victims to poison their ARP cache. Once the cac
 has been poisoned the victims will send all packets to the attacker which, in
 turn, can modify and forward them to the real destination.
 .Sp
-In silent mode (-z option) only the first target is selected, if you want to
-poison multiple target in silent mode use the -j option to load a list from a
+In silent mode (\-z option) only the first target is selected, if you want to
+poison multiple target in silent mode use the \-j option to load a list from a
 file.
 .Sp
 You can select empty targets and they will be expanded as 'ANY' (all the hosts in
@@ -258,7 +257,7 @@ an arp watcher can be in place).
 .Sp
 Example:
 .Sp
-the targets are: /10.0.0.1-5/ /10.0.0.15-20/
+the targets are: /10.0.0.1\-5/ /10.0.0.15\-20/
 .br
 and the host list is: 10.0.0.1 10.0.0.3 10.0.0.16 10.0.0.18
 .Sp
@@ -294,7 +293,7 @@ NOTE: to restrict the redirection to a given target, specify it as a TARGET
 .Sp
 Example:
 .Sp
--M icmp:00:11:22:33:44:55/10.0.0.1
+\-M icmp:00:11:22:33:44:55/10.0.0.1
 .Sp
 will redirect all the connections that pass thru that gateway.
 
@@ -331,11 +330,11 @@ is the gateway until the lease expires...
 .Sp
 Example:
 .Sp
--M dhcp:192.168.0.30,35,50-60/255.255.255.0/192.168.0.1
+\-M dhcp:192.168.0.30,35,50\-60/255.255.255.0/192.168.0.1
 .br
 reply to DHCP offer and request.
 .Sp
--M dhcp:/255.255.255.0/192.168.0.1
+\-M dhcp:/255.255.255.0/192.168.0.1
 .br
 reply only to DHCP request.
 
@@ -374,7 +373,7 @@ You can perform either HALF or FULL DUPLEX mitm according to target selection.
 NOTE: Use this mitm method only on ethernet switches. Use it carefully, 
 it could produce performances loss or general havoc.
 .Sp
-NOTE: You can NOT use this method in only-mitm mode (-o flag), because it
+NOTE: You can NOT use this method in only-mitm mode (\-o flag), because it
 hooks the sniffing engine, and you can't use interactive data injection.
 .Sp
 NOTE: It could be dangerous to use it in conjunction with other mitm methods.
@@ -400,7 +399,7 @@ You will intercept and visualize all the traffic for 10.0.0.1.
 .RE
 
 .TP
-\fB\-o\fR, \fB\-\-only-mitm\fR
+\fB\-o\fR, \fB\-\-only\-mitm\fR
 This options disables the sniffing thread and enables only the mitm attack.
 Useful if you want to use ettercap to perform mitm attacks and another sniffer
 (such as ethereal) to sniff the traffic. Keep in mind that the packets are not
@@ -457,7 +456,7 @@ NOTE: dump file collect ALL the packets disregarding the TARGET. This is done
 because you may want to log even protocols not supported by ettercap, so you
 can analyze them with other tools.
 .Sp
-TIP: you can use the -w option in conjunction with the -r one. This way you
+TIP: you can use the \-w option in conjunction with the \-r one. This way you
 will be able to filter the payload of the dumped packets or decrypt
 WEP-encrypted WiFi traffic and dump them to another file.
 
@@ -479,7 +478,7 @@ ettercap log files.
 .Sp
 example:
 .Sp
-ettercap -Tq -L dumpfile -r pcapfile
+ettercap \-Tq \-L dumpfile \-r pcapfile
 
 .TP
 \fB\-s\fR, \fB\-\-script <COMMANDS>\fR
@@ -490,9 +489,9 @@ command will sleep for x seconds.
 .Sp
 example:
 .Sp
-ettercap -T -s 'lq'  will print the list of the hosts and exit
+ettercap \-T \-s 'lq'  will print the list of the hosts and exit
 .br
-ettercap -T -s 's(300)olqq'  will collect the infos for 5 minutes, print the
+ettercap \-T \-s 's(300)olqq'  will collect the infos for 5 minutes, print the
 list of the local profiles and exit
 
 
@@ -590,7 +589,7 @@ forward packets itself. This option prevent to do that, so the responsibility
 of ip forwarding is left to the kernel.
 .br
 This options is useful if you want to run multiple ettercap instances. You will
-have one instance (the one without the -u option) forwarding the packets, and
+have one instance (the one without the \-u option) forwarding the packets, and
 all the other instances doing their work without forwarding them. Otherwise you will
 get packet duplicates.
 .br
@@ -606,26 +605,26 @@ This is also useful if you want to run ettercap on the gateway. It will not
 disable the forwarding and the gateway will correctly route the packets.
 
 .TP
-\fB\-j\fR, \fB\-\-load-hosts <FILENAME>\fR
-It can be used to load a hosts list from a file created by the -k option. (see below)
+\fB\-j\fR, \fB\-\-load\-hosts <FILENAME>\fR
+It can be used to load a hosts list from a file created by the \-k option. (see below)
 
 .TP
-\fB\-k\fR, \fB\-\-save-hosts <FILENAME>\fR
+\fB\-k\fR, \fB\-\-save\-hosts <FILENAME>\fR
 Saves the hosts list to a file. Useful when you have many hosts and you don't want to
 do an ARP storm at startup any time you use ettercap. Simply use this options and dump
-the list to a file, then to load the information from it use the -j <filename> option.
+the list to a file, then to load the information from it use the \-j <filename> option.
 
 .TP
 \fB\-P\fR, \fB\-\-plugin <PLUGIN>\fR
 Run the selected PLUGIN. Many plugins need target specification, use TARGET as
 always.
 .br
-In console mode (-C option), standalone plugins are executed and then the
+In console mode (\-C option), standalone plugins are executed and then the
 application exits. Hook plugins are activated and the normal sniffing is
 performed.
 .br
 To have a list of the available external plugins use "list" (without quotes) as
-plugin name (e.g. ./ettercap -P list).
+plugin name (e.g. ./ettercap \-P list).
 .Sp
 NOTE: you can also activate plugins directly from the interfaces (always press
 "h" to get the inline help)
@@ -643,7 +642,7 @@ Any number of filters can be loaded by specifying the option multiple times;
 packets are passed through each filter in the order specified on the command line.
 You can also load a script without enabling it by appending :0 to the filename.
 .br
-NOTE: these filters are different from those set with --pcapfilter. An ettercap
+NOTE: these filters are different from those set with \-\-pcapfilter. An ettercap
 filter is a content filter and can modify the payload of a packet before
 forwarding it. Pcap filter are used to capture only certain packets.
 .br
@@ -653,7 +652,7 @@ ettercap will not recalculate checksums, nor split packets exceeding the mtu
 (snaplen) nor anything like that.
 
 .TP
-\fB\-W\fR, \fB\-\-wep-key <KEY>\fR
+\fB\-W\fR, \fB\-\-wep\-key <KEY>\fR
 You can specify a WEP key to decrypt WiFi packets. Only the packets decrypted
 successfully will be passed to the decoders stack, the others will be skipped
 with a message.
@@ -664,11 +663,11 @@ passphrase). KEY can be a string or an escaped hex sequences.
 .Sp
 example:
 .br
---wep-key 128:p:secret
+\-\-wep\-key 128:p:secret
 .br
---wep-key 128:s:ettercapwep0
+\-\-wep\-key 128:s:ettercapwep0
 .br
---wep-key '64:s:\\x01\\x02\\x03\\x04\\x05'
+\-\-wep\-key '64:s:\\x01\\x02\\x03\\x04\\x05'
 
 .TP
 \fB\-a\fR, \fB\-\-config <CONFIG>\fR
@@ -684,7 +683,7 @@ This is useful if you have many preconfigured files for different situations.
 \fB\-e\fR, \fB\-\-regex <REGEX>\fR
 Handle only packets that match the regex.
 .br
-This option is useful in conjunction with -L. It logs only packets that match
+This option is useful in conjunction with \-L. It logs only packets that match
 the posix regex REGEX.
 .br
 It impacts even the visualization of the sniffed packets. If it is set only
@@ -758,7 +757,7 @@ because the reply was previously sniffed. think about it as a passive dns
 resolution for free... ;)
 
 .TP
-\fB\-E\fR, \fB\-\-ext-headers\fR
+\fB\-E\fR, \fB\-\-ext\-headers\fR
 Print extended headers for every displayed packet. (e.g. mac addresses)
 
 .TP
@@ -769,11 +768,11 @@ but you don't want to be flooded with dissectors messages. Useful when using
 plugins because the sniffing process is always active, it will print all the
 collected infos, with this option you can suppress these messages.
 .br
-NOTE: this options automatically sets the -q option.
+NOTE: this options automatically sets the \-q option.
 .Sp
 example:
 .Sp
-ettercap -TzQP finger /192.168.0.1/22
+ettercap \-TzQP finger /192.168.0.1/22
 
 
 
@@ -793,16 +792,16 @@ privileges since the log file is opened in the startup phase (with high
 privs). But if you enable the log option while ettercap is already started, you
 have to be in a directory where uid = 65535 or uid = EC_UID can write.
 .Sp
-NOTE: the logfiles can be compressed with the deflate algorithm using the -c
+NOTE: the logfiles can be compressed with the deflate algorithm using the \-c
 option.
 
 .TP
-\fB\-l\fR, \fB\-\-log-info <LOGFILE>\fR
-Very similar to -L but it logs only passive information + users and passwords
+\fB\-l\fR, \fB\-\-log\-info <LOGFILE>\fR
+Very similar to \-L but it logs only passive information + users and passwords
 for each host. The file will be named LOGFILE.eci
 
 .TP
-\fB\-m\fR, \fB\-\-log-msg <LOGFILE>\fR
+\fB\-m\fR, \fB\-\-log\-msg <LOGFILE>\fR
 It stores in <LOGFILE> all the user messages printed by ettercap. This can be
 useful when you are using ettercap in daemon mode or if you want to track down
 all the messages. Indeed, some dissectors print messages but their
@@ -815,7 +814,7 @@ Compress the logfile with the gzip algorithm while it is dumped. etterlog(8) is
 capable of handling both compressed and uncompressed log files.
 
 .TP
-\fB\-o\fR, \fB\-\-only-local\fR
+\fB\-o\fR, \fB\-\-only\-local\fR
 Stores profiles information belonging only to the LAN hosts.
 .Sp
 NOTE: this option is effective only against the profiles collected in memory.
@@ -823,7 +822,7 @@ While logging to a file ALL the hosts are logged. If you want to split them, use
 the related etterlog(8) option.
 
 .TP
-\fB\-O\fR, \fB\-\-only-remote\fR
+\fB\-O\fR, \fB\-\-only\-remote\fR
 Stores profiles information belonging only to remote hosts.
 
 
@@ -836,8 +835,8 @@ Stores profiles information belonging only to remote hosts.
 Connects to the ettercap website (ettercap.sf.net) and retrieve the latest
 databases used by ettercap.
 .br
-If you want only to check if an update is available, prepend the -z option.
-The order does matter: ettercap -zU
+If you want only to check if an update is available, prepend the \-z option.
+The order does matter: ettercap \-zU
 .Sp
 .B SECURITY NOTE: 
 The updates are not signed so an attacker may poison your DNS server 
@@ -860,50 +859,50 @@ prints the help screen with a short summary of the available options.
 .SH EXAMPLES
 Here are some examples of using ettercap.
 .TP
-.B ettercap -Tp
+.B ettercap \-Tp
 .Sp
 Use the console interface and do not put the interface in promisc mode. You
 will see only your traffic.
 
 .TP
-.B ettercap -Tzq
+.B ettercap \-Tzq
 .Sp
 Use the console interface, do not ARP scan the net and be quiet. The packet
 content will not be displayed, but user and passwords, as well as other
 messages, will be displayed.
 
 .TP
-.B ettercap -T -j /tmp/victims -M arp /10.0.0.1-7/ /10.0.0.10-20/
+.B ettercap \-T \-j /tmp/victims \-M arp /10.0.0.1\-7/ /10.0.0.10\-20/
 .Sp
 Will load the hosts list from /tmp/victims and perform an ARP poisoning attack
 against the two target. The list will be joined with the target and the
 resulting list is used for ARP poisoning.
 
 .TP
-.B ettercap -T -M arp // //
+.B ettercap \-T \-M arp // //
 .Sp
 Perform the ARP poisoning attack against all the hosts in the LAN. BE CAREFUL
 !!
 
 .TP
-.B ettercap -T -M arp:remote /192.168.1.1/ /192.168.1.2-10/
+.B ettercap \-T \-M arp:remote /192.168.1.1/ /192.168.1.2\-10/
 .Sp
 Perform the ARP poisoning against the gateway and the host in the lan between 2
 and 10. The 'remote' option is needed to be able to sniff the remote traffic
 the hosts make through the gateway.
 
 .TP
-.B ettercap -Tzq //110
-.SP
+.B ettercap \-Tzq //110
+.Sp
 Sniff only the pop3 protocol from every hosts.
 
 .TP
-.B ettercap -Tzq /10.0.0.1/21,22,23 
+.B ettercap \-Tzq /10.0.0.1/21,22,23 
 .Sp
 Sniff telnet, ftp and ssh connections to 10.0.0.1.
 
 .TP
-.B ettercap -P list
+.B ettercap \-P list
 .Sp
 Prints the list of all available plugins
 
@@ -928,10 +927,8 @@ Marco Valleri (NaGA) <naga@antifork.org>
 http://ettercap.sourceforge.net/download/
 
 
-.SH CVS
-cvs -d:pserver:anonymous@cvs.ettercap.sf.net:/cvsroot/ettercap login
-.br
-cvs -d:pserver:anonymous@cvs.ettercap.sf.net:/cvsroot/ettercap co ettercap_ng
+.SH GIT
+git clone \-b ettercap_rc git://github.com/Ettercap/ettercap.git ettercap_rc 
 
 
 .SH BUGS
diff --git a/man/ettercap_curses.8.in b/man/ettercap_curses.8.in
index c9f48df..8e5eb5c 100644
--- a/man/ettercap_curses.8.in
+++ b/man/ettercap_curses.8.in
@@ -22,7 +22,7 @@
 ..
 .TH ETTERCAP-CURSES "8" "" "ettercap @VERSION@"
 .SH NAME
-.B ettercap @VERSION@ \- Man page for the Ncurses GUI.
+ettercap \- Man page for the Ncurses GUI.
 
 .SH GENERAL DESCRIPTION
 The curses GUI is quite simple and intuitive.
@@ -63,9 +63,9 @@ To use the ncurses GUI you have to:
 .Sp
 - compile ettercap with ncurses support (obviously)
 .br
-- run it with the -C flag 
+- run it with the \-C flag 
 .Sp
-Passing the -C flag is sufficient, but if you want you can pass other flags
+Passing the \-C flag is sufficient, but if you want you can pass other flags
 that will be automatically set for the ncurses GUI. You will be able to
 override them using the menu to change the options.
 
@@ -311,16 +311,16 @@ Unload the filter and stop filtering the connections.
 .B Log all packets and infos...
 Given a file name, it will create two files: filename.eci (for information
 about hosts) and filename.ecp (for all the interesting packets). This is the
-same as the -L option.
+same as the \-L option.
 .TP
 .B Log only infos...
-This is used only to sniff information about hosts (same as the -l option).
+This is used only to sniff information about hosts (same as the \-l option).
 .TP
 .B Stop logging info
 Come on... it is self explanatory.
 .TP
 .B Log user messages...
-Will log all the messages appearing in the bottom window (same as -m option).
+Will log all the messages appearing in the bottom window (same as \-m option).
 .TP
 .B Compressed file
 Asterisk-option to control whether or not the logfile should be compressed.
diff --git a/man/ettercap_plugins.8.in b/man/ettercap_plugins.8.in
index 693abff..3c1893b 100644
--- a/man/ettercap_plugins.8.in
+++ b/man/ettercap_plugins.8.in
@@ -22,12 +22,12 @@
 ..
 .TH ETTERCAP-PLUGINS "8" "" "ettercap @VERSION@"
 .SH NAME
-.B ettercap-plugins @VERSION@ \- A collection of plugins for ettercap
+ettercap-plugins \- A collection of plugins for ettercap
 
 .SH DESCRIPTION
 Ettercap(8) supports loadable modules at runtime. They are called plugins and
 they come within the source tarball. They are automatically compiled if your
-system supports them or until you specify the --disable-plugins option to the
+system supports them or until you specify the \-\-disable\-plugins option to the
 configure script.
 .br
 Some of older ettercap plugins (roper, banshee, and so on) have not been ported 
@@ -39,7 +39,7 @@ If you use interactive mode, most plugins need to "Start Sniff" before using the
 .TP
 To have a list of plugins installed in your system do that command:
 .Sp
-.I ettercap -P list
+.I ettercap \-P list
 
 .LP
 .LP
@@ -54,7 +54,7 @@ If you build the initial host list the plugin will run more accurately.
 .Sp
 .I example :
 .Sp
-ettercap -TQP arp_cop // 
+ettercap \-TQP arp_cop // 
 
 
 .TP 
@@ -110,7 +110,7 @@ This plugin is based on the original Naptha DoS attack
 .Sp
 .I example :
 .Sp
-ettercap -TQP dos_attack
+ettercap \-TQP dos_attack
 
 .TP 
 .B dummy
@@ -126,9 +126,9 @@ wants to talk to. It can also help you finding addresses in an unknown LAN.
 .Sp
 .I example :
 .Sp
-ettercap -TQzP find_conn 
+ettercap \-TQzP find_conn 
 .Sp
-ettercap -TQu -i eth0 -P find_conn 
+ettercap \-TQu \-i eth0 \-P find_conn 
 
 
 .TP 
@@ -156,9 +156,9 @@ You can run it either from the command line or from the proper menu.
 .Sp
 .I example :
 .Sp
-ettercap -TQP find_ip // 
+ettercap \-TQP find_ip // 
 .Sp
-ettercap -TQP find_ip /192.168.0.1-254/ 
+ettercap \-TQP find_ip /192.168.0.1\-254/ 
 
 
 .TP 
@@ -175,9 +175,9 @@ ports, all the ports will be tested on all the IPs.
 .Sp
 .I example :
 .Sp
-ettercap -TzP finger /192.168.0.1/22
+ettercap \-TzP finger /192.168.0.1/22
 .br
-ettercap -TzP finger /192.168.0.1-50/22,23,25
+ettercap \-TzP finger /192.168.0.1\-50/22,23,25
 
 
 .TP 
@@ -191,7 +191,7 @@ database. Thank you very much.
 .Sp
 .I example :
 .Sp
-ettercap -TzP finger_submit
+ettercap \-TzP finger_submit
 
 .TP 
 .B gre_relay
@@ -220,7 +220,7 @@ have a valid host list before launching this plugin.
 .Sp
 .I example :
 .Sp
-ettercap -TP gw_discover /192.168.0.1-50/
+ettercap \-TP gw_discover /192.168.0.1\-50/
 
 .TP 
 .B isolate
@@ -237,9 +237,9 @@ be blocked to the victim.
 .Sp
 .I examples :
 .Sp
-ettercap -TzqP isolate /192.168.0.1/ //
+ettercap \-TzqP isolate /192.168.0.1/ //
 .br
-ettercap -TP isolate /192.168.0.1/ /192.168.0.2-30/
+ettercap \-TP isolate /192.168.0.1/ /192.168.0.2\-30/
 
 
 .TP 
@@ -252,9 +252,9 @@ accurate.
 .Sp
 .I example :
 .Sp
-ettercap -TQP link_type /192.168.0.1/
+ettercap \-TQP link_type /192.168.0.1/
 .br
-ettercap -TQP link_type //
+ettercap \-TQP link_type //
 
 .TP 
 .B pptp_chapms1
@@ -304,7 +304,7 @@ It is useful only on ethernet switches.
 .Sp
 .I example :
 .Sp
-ettercap -TP rand_flood
+ettercap \-TP rand_flood
 
 
 .TP 
@@ -325,9 +325,9 @@ in the targets' lists, it replies with attacker's MAC address.
 .Sp
 .I example :
 .Sp
-ettercap -TQzP reply_arp /192.168.0.1/
+ettercap \-TQzP reply_arp /192.168.0.1/
 .br
-ettercap -TQzP reply_arp //
+ettercap \-TQzP reply_arp //
 
 .TP
 .B repoison_arp
@@ -345,7 +345,7 @@ arp-poisoning mitm method.
 .Sp
 .I example :
 .Sp
-ettercap -T -M arp:remote -P repoison_arp /192.168.0.10-20/ /192.168.0.1/
+ettercap \-T \-M arp:remote \-P repoison_arp /192.168.0.10\-20/ /192.168.0.1/
 
 .TP 
 .B scan_poisoner
@@ -364,7 +364,7 @@ You can't perform this active test in unoffensive mode.
 .Sp
 .I example :
 .Sp
-ettercap -TQP scan_poisoner //
+ettercap \-TQP scan_poisoner //
 
 .TP 
 .B search_promisc
@@ -379,9 +379,9 @@ arp request.
 .Sp
 .I example :
 .Sp
-ettercap -TQP search_promisc /192.168.0.1/
+ettercap \-TQP search_promisc /192.168.0.1/
 .br
-ettercap -TQP search_promisc //
+ettercap \-TQP search_promisc //
 
 
 .TP 
@@ -416,7 +416,7 @@ decrease your MAC address before running it.
 .Sp
 .I example :
 .Sp
-ettercap -TP stp_mangler 
+ettercap \-TP stp_mangler 
 
 .SH "SEE ALSO"
 .I "ettercap(8)"
diff --git a/man/etterfilter.8.in b/man/etterfilter.8.in
index 90ed203..ee1da10 100644
--- a/man/etterfilter.8.in
+++ b/man/etterfilter.8.in
@@ -22,7 +22,7 @@
 ..
 .TH ETTERFILTER "8" "" "ettercap @VERSION@"
 .SH NAME
-.B etterfilter @VERSION@ \- Filter compiler for ettercap content filtering engine
+etterfilter \- Filter compiler for ettercap content filtering engine
 
 .SH SYNOPSIS
 .B etterfilter
@@ -52,7 +52,7 @@ in a human readable form all the instructions contained in it. It is a sort of
 .TP
 \fB\-d\fR, \fB\-\-debug\fR
 prints some debug messages during the compilation. Use it more than once to
-increase the debug level ( etterfilter -ddd ... ).
+increase the debug level ( etterfilter \-ddd ... ).
                            
 .TP
 \fB\-w\fR, \fB\-\-suppress-warnings\fR
@@ -226,7 +226,7 @@ will swap the value of var1 and var2).
 NOTE: The pcre support is optional in ettercap and will be enabled only if you
 have the libpcre installed.
 The compiler will warn you if you try to compile a filter that contains
-pcre expressions but you don't have libpcre. Use the -w option to suppress the
+pcre expressions but you don't have libpcre. Use the \-w option to suppress the
 warning.
 .Sp
 example:
@@ -266,7 +266,7 @@ inject("./fake_packet")
 this function dumps in the file 'where' the buffer 'what'. No information is
 stored about the packet, only the payload is dumped. So you will see the stream
 in the file. If you want to log packets in a more enhanced mode, you need to
-use the ettercap -L option and analyze it with etterlog(8).
+use the ettercap \-L option and analyze it with etterlog(8).
 .br
 The file 'where' must be writable to the user EC_UID (see etter.conf(5)).
 .Sp
@@ -332,7 +332,7 @@ exit()
 .SH EXAMPLES
 Here are some examples of using etterfilter.
 .TP
-.B etterfilter filter.ecf -o filter.ef
+.B etterfilter filter.ecf \-o filter.ef
 .Sp
 Compiles the source filter.ecf into a binary filter.ef
 
diff --git a/man/etterlog.8.in b/man/etterlog.8.in
index 854a6c2..2f545ed 100644
--- a/man/etterlog.8.in
+++ b/man/etterlog.8.in
@@ -22,7 +22,7 @@
 ..
 .TH ETTERLOG "8" "" "ettercap @VERSION@"
 .SH NAME
-.B etterlog @VERSION@ \- Log analyzer for ettercap log files
+etterlog \- Log analyzer for ettercap log files
 
 .SH SYNOPSIS
 .B etterlog
@@ -31,7 +31,7 @@
 
 .SH DESCRIPTION
 Etterlog is the log analyzer for logfiles created by ettercap. It can handle
-both compressed (created with -Lc) or uncompressed logfiles. With this tool you
+both compressed (created with \-Lc) or uncompressed logfiles. With this tool you
 can manipulate binary files as you like and you can print data in
 different ways all the times you want (in contrast with the previous logging
 system which was used to dump in a single static manner).
@@ -62,7 +62,7 @@ useless.
 .Sp
 TIP: you can search for a particular host by using the following command:
 .Sp
-etterlog -c logfile.ecp | grep 10.0.0.1
+etterlog \-c logfile.ecp | grep 10.0.0.1
 
 .TP
 \fB\-f\fR, \fB\-\-filter <TARGET>\fR
@@ -98,50 +98,50 @@ CONNECTION is in the form PROTO:SOURCE:DEST. SOURCE and DEST are in the form IP:
 .Sp
 example:
 .Sp
-etterlog -F TCP:10.0.0.23:3318:198.182.196.56:80
+etterlog \-F TCP:10.0.0.23:3318:198.182.196.56:80
 
 .TP
-\fB\-s\fR, \fB\-\-only-source\fR
+\fB\-s\fR, \fB\-\-only\-source\fR
 Display only packets that are sent by the source of the selected CONNECTION.
-This option makes sense only in conjunction with the -F option. 
+This option makes sense only in conjunction with the \-F option. 
 .Sp
 TIP: if you want to save a file transferred in an HTTP or FTP connection, you can
 use the following command:
 .Sp
-etterlog -B -s -n -F TCP:10.0.0.1:20:10.0.0.2:35426 logfile.ecp >
+etterlog \-B \-s \-n \-F TCP:10.0.0.1:20:10.0.0.2:35426 logfile.ecp >
 example.tar.gz
   
 .TP
-\fB\-d\fR, \fB\-\-only-dest\fR
-Same as --only-source but it filters on the destination host.
+\fB\-d\fR, \fB\-\-only\-dest\fR
+Same as \-\-only\-source but it filters on the destination host.
 
 
 .TP
-\fB\-n\fR, \fB\-\-no-headers\fR
+\fB\-n\fR, \fB\-\-no\-headers\fR
 Do not print the header of each packet. This option is useful if you want to
-save a file in binary format (-B option). Without the headers you can redirect
+save a file in binary format (\-B option). Without the headers you can redirect
 the output to a file and you will get the original stream.
 .Sp
 NOTE: the time stamp in the header is in the form: Thu Mar 27 23:03:31 2003
 [169396], the value in the square brackets is expressed in microseconds
 
 .TP
-\fB\-m\fR, \fB\-\-show-mac\fR
+\fB\-m\fR, \fB\-\-show\-mac\fR
 In the headers show also the mac addresses corresponding to the ip
 addresses.
 
 .TP
 \fB\-k\fR, \fB\-\-color\fR
-If used in conjunction with -F it displays the source and dest of the connection
+If used in conjunction with \-F it displays the source and dest of the connection
 using different colors. If used with a LOG_INFO file it prints LAN hosts in green, 
 REMOTE hosts in blue and GATEWAYS in red.
 
 .TP
-\fB\-l\fR, \fB\-\-only-local\fR
+\fB\-l\fR, \fB\-\-only\-local\fR
 Used displaying an INFO file, it displays information only about local hosts.
   
 .TP
-\fB\-L\fR, \fB\-\-only-remote\fR
+\fB\-L\fR, \fB\-\-only\-remote\fR
 Used displaying an INFO file, it displays information only about remote hosts.
 
 
@@ -166,12 +166,12 @@ user/pass couples collected across all hosts.
 .TP
 \fB\-p\fR, \fB\-\-passwords\fR
 Print only the collected account information for each host. This prevents 
-the huge profile output. It can be used in conjunction with the -u option 
+the huge profile output. It can be used in conjunction with the \-u option 
 to filter the users. An asterisk '*' used in front of an account represents
 a failed login attempt.
 
 .TP
-\fB\-i\fR, \fB\-\-show-client\fR
+\fB\-i\fR, \fB\-\-show\-client\fR
 Show the client ip address when displaying the collected users and
 passwords. It may be useful when ACLs are in place.
 
@@ -189,12 +189,12 @@ usernames and passwords of a client.
 \fB\-C\fR, \fB\-\-concat\fR
 Use this option to concatenate two (or more) files into one single file. This
 is useful if you have collected ettercap log files from multiple sources and
-want to have an unified report. The output file must be specified with the -o
+want to have an unified report. The output file must be specified with the \-o
 option and the input files are listed as normal arguments.
 .Sp
 example:
 .br 
-etterlog -C -o outfile input1 input2 input3
+etterlog \-C \-o outfile input1 input2 input3
 
 .TP
 \fB\-o\fR, \fB\-\-outfile <FILE>\fR
@@ -249,7 +249,7 @@ This is the title, but the following will not be displayed.
 .TP
 \fB\-U\fR, \fB\-\-utf8 <encoding>\fR
 Print the packets in UTF-8 format. The <encoding> parameter specifies the
-encoding to be used while performing the conversion. Use the `iconv --list`
+encoding to be used while performing the conversion. Use the `iconv \-\-list`
 command to obtain a list of all supported encodings.
 
 .TP
@@ -279,68 +279,68 @@ Print the help screen with a short summary of the available options.
 .SH EXAMPLES
 Here are some examples of using etterlog.
 .TP
-.B etterlog -k -l dump.eci
+.B etterlog \-k \-l dump.eci
 .Sp
 Displays information about local hosts in different colors.
 
 .TP
-.B etterlog -X dump.ecp
+.B etterlog \-X dump.ecp
 .Sp
 Prints packets in HEX mode with full headers.
 
 .TP
-.B etterlog -c dump.ecp
+.B etterlog \-c dump.ecp
 .Sp
 Displays the list of connections logged in the file.
 
 .TP
-.B etterlog -Akn -F TCP:10.0.0.1:13423:213.203.143.52:6666 dump.ecp
+.B etterlog \-Akn \-F TCP:10.0.0.1:13423:213.203.143.52:6666 dump.ecp
 .Sp
 Displays the IRC traffic made by 10.0.0.1 in ASCII mode, without headers
 information and in colored mode.
 
 .TP
-.B etterlog -H -t tcp -f //80 dump.ecp
+.B etterlog \-H \-t tcp \-f //80 dump.ecp
 .Sp
 Dumps all HTTP traffic and strips html tags.
 
 .TP
-.B etterlog -Z -r -f /10.0.0.2/22 dump.ecp
+.B etterlog \-Z \-r \-f /10.0.0.2/22 dump.ecp
 .Sp
 Displays only the headers of all connections except ssh on host 10.0.0.2
 
 .TP
-.B etterlog -A -e 'user' -f //110 dump.ecp
+.B etterlog \-A \-e 'user' \-f //110 dump.ecp
 .Sp
 Displays only POP packets containing the 'user' regexp (case insensitive).
 
 .TP
-.B etterlog -u root dump.eci
+.B etterlog \-u root dump.eci
 .Sp
 Displays information about all the accounts of the user 'root'.
 
 .TP
-.B etterlog -e Apache dump.eci
+.B etterlog \-e Apache dump.eci
 .Sp
 Displays information about all the hosts running 'Apache'.
 
 .TP
-.B etterlog -e Linux dump.eci
+.B etterlog \-e Linux dump.eci
 .Sp
 Displays information about all the hosts with the 'Linux' operating system.
 
 .TP
-.B etterlog -t tcp -f //110 dump.eci
+.B etterlog \-t tcp \-f //110 dump.eci
 .Sp
 Displays information about all the hosts with the tcp port 110 open.
 
 .TP
-.B etterlog -t udp dump.eci
+.B etterlog \-t udp dump.eci
 .Sp
 Displays information about all the hosts with at least one UDP port open.
 
 .TP
-.B etterlog -B -s -n -F TCP:10.0.0.1:20:10.0.0.2:35426 logfile.ecp > example.tar.gz
+.B etterlog \-B \-s \-n \-F TCP:10.0.0.1:20:10.0.0.2:35426 logfile.ecp > example.tar.gz
 .Sp
 Dumps in binary form the data sent by 10.0.0.1 over the data port of FTP. Since
 the headers are omitted, you will get the file as it was.
